portmaster/guides/contribute.md

```diff
@@ -1,13 +1,48 @@
 ---
 title: How To Contribute
+toc_max_header: 3
 layout: base
 ---
 
 ###### Thanks For Participating!
 
-Your contributions will have the highest impact when aligned with Safing's current goals and focus. As a result, these are the current scopes where we are super thankful for your participation:
+It is amazing to see you love Portmaster so much that you are checking out this guide. And honestly, as a small team **your contribution means the world to us.** We could not do it without you!
 
-## Use the Portmaster
+## Spread the Word
+
+> With [version 1.0](https://safing.io/blog/2022/10/27/portmaster-reaches-1.0/), Portmaster should be recommended as one of the first steps anybody should take within their privacy journey.
+
+<div class="notification-warning">
+  <i class="fas fa-fire" style="width:4em;font-size:2em;color:white;text-align:center;padding-top:0.2em;padding-right:0.5em;"></i>
+  <p>
+    As new software - too few people know about Portmaster. <br/> This is where your impact will be biggest!
+  </p>
+</div>
+
+
+### Ask Your Favorite Creators For Reviews
+
+Think of your favorite creators and shows. It could be YouTubers, Newsletters, blogs, guides, lists, radio shows, podcasts and so on. Do these creators know about Portmaster already? Would they enjoy sharing a review on their show? Give them a ping and ask them what they think about Portmaster. This could domino effect into hundreds of thousands of people getting better privacy!
+
+### Do You Have An Audience?
+
+Are you a creator or do you give recommendations? No matter what you think of Portmaster - we'd love for you to have a look and tell your audience what you think!
```

Comment:
I like it better with an if at the front, but I don't mind going either way.

If you are a creator or someone who gives recommendations, no matter.... ( you could also swap it then and put the No matter what as a second sentence behind)